### ISSUE TITLE
Landing page misses important segments when zoomed out

### ISSUE THREAD

**Issue by Robinlovelace** [nptscot/nptscot.github.io#244]
this has happened lots, see Edinburghs network is a bit missing...

![image](https://github.com/user-attachments/assets/352f6b7f-d7a7-4703-ac36-90e2aa690015)

**Comment by Robinlovelace** [comment]
Closing as minor and not reproducible.

**Comment by Robinlovelace** [comment]
Seems to be fixed on dev version:

![image](https://github.com/user-attachments/assets/6a8b03c9-724e-48c0-9729-652c1651fcf5)

Source: https://nptscot.github.io/#/rnet/#8/55.953/-3.138

But not on production version: https://nptscot.github.io/#/rnet/#8/55.953/-3.138

Can some please check, to ensure it's not something with my setup and report back? If it *is* fixed on test website that's great, we can update production and close.

Cc @wangzhao0217, @Hussein-Mahfouz, @juanfonsecaLS1, @mvl22 and @mem48 if you could try clicking those links and reporting back that would be greatly appreciated.

**Comment by mem48** [comment]
I suspect the tile is too big and maplibre is failing somehow. No obvious error message. But doesn't seem to affect simplified network. I'd suggest dropping some of the low flow roads when zoomed out.

**Comment by Robinlovelace** [comment]
Yes, I've figured it out: it's dropping sections near-randomly by location now. Will work on a fix sooon.

**Comment by Robinlovelace** [comment]
Tracked in https://github.com/nptscot/npt/issues/561
